src/chess/bitboard.h

```diff
@@ -240,10 +264,15 @@ class Move {
   }
 
  private:
-  BoardSquare from_;
-  BoardSquare to_;
-  Promotion promotion_ = Promotion::None;
-  bool castling_ = false;
+  uint16_t data_ = 0;  // First bit castling, next 3 promotion, next 6
+  // fromsquare, next 6 tosquare.
+
+  enum MASKS_ : uint16_t {
```

Comment:
`MASKS_` is a type name, should be mixed case: `Masks` and `BitMasks` etc.

Reply:
Done